types/formidable/index.d.ts

```diff
@@ -29,6 +32,20 @@ export declare class IncomingForm extends events.EventEmitter {
     parse(req: http.IncomingMessage, callback?: (err: any, fields: Fields, files: Files) => any): void;
 }
 
+export declare interface IncomingFormOptions {
+    encoding?: string;
```

Comment:
this is probably same like in encoding in Crypto,
so optionally, can this be type of crypto.Encoding? (I'm not sure if crypto exports this for consumption), can you check this? This would improve experience, thx!
```ts
type Encoding = BinaryToTextEncoding | CharacterEncoding | LegacyCharacterEncoding;
```